documents/service.go

```diff
@@ -238,6 +243,11 @@ func (s service) ReceiveAnchoredDocument(ctx context.Context, model Model, colla
 		return errors.NewTypedError(ErrDocumentInvalid, err)
 	}
 
+	// set the status to committed since the document is anchored already.
```

Comment:
I think we need as well to change the state to `Committed` on the sender node?

Reply:
We cant, sender node can be a different implementation altogether. Status should be within the local storage IMO else protocol will know what is a status :-D